app/Http/Controllers/CategoryController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\CategoryStoreRequest;
+use App\Http\Requests\CategoryUpdateRequest;
+use App\Models\Category;
+use Illuminate\Http\Request;
+
+class CategoryController extends Controller
+{
+    public function index()
+    {
+        return view('categories.index')
+            ->with('categories', Category::orderBy('id', 'desc')->paginate(15));
```

Comment:
What if you would like to paginate not for 15? Then you will have to go through every controller and change it. Just set number in model file `protected $perPage = 15;`